lib/jwt/verify.rb

```diff
@@ -38,12 +38,12 @@ def verify_aud
     end
 
     def verify_expiration
-      return unless @payload.include?('exp')
+      return unless @payload.key?('exp')
```

Comment:
What if there would be a method on this class, something like 

```
def contains_key?(payload, key)
  payload.respond_to?(:key?) && payload.key?(key)
end
```

Then this row would be
```
return unless contains_key?(@payload, 'exp')
```